content/en/about/faq/security/is-ztunnel-a-SPOF.md

```diff
@@ -0,0 +1,6 @@
+---
+title: Is ztunnel a SPOF?
+weight: 25
+---
+
+This misunderstanding arises primarily from concerns about the Ztunnel failures are scoped to a single node, which is considered a fallible component in a cluster. It behaves the same as other node-critical infrastructure running on every cluster such as the Linux kernel, container runtime, etc. In a properly designed system, node outages do not lead to cluster outages.
```

Comment:
the link to the blog and other details seem to be removed? why?

Reply:
First of all link should be of relative path. That's what confusing me little bit, that should I keep or remove it. I just removed it because the description seems enough to me. Should I raise another PR for it.